fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1039,6 +1040,32 @@ public void testIngestOnSubtree() throws Exception {
 
     }
 
+    /**
+     * @author griffinj
+     * Ensure that the objects cannot be pairtree child resources
+     * @see <a href="https://jira.duraspace.org/browse/FCREPO-1505">FCREPO-1505</a>
+     *
+     */
+    @Test
+    public void testIngestOnPairtree() throws Exception {
+        HttpPost method = postObjMethod("");
+        HttpResponse response = client.execute(method);
+        final String location = response.getFirstHeader("Location").getValue();
+
+        // Get the segments from the URL
+        final String[] segments = location.split("/");
+
+        LOGGER.info("POSTing to " + serverAddress + "/" + segments[4] + "/");
```

Comment:
@ajs6f My apologies for this.  I have attempted to resolve this by undertaking the approach utilized  within [FedoraExportIT#shouldRoundTripOnePairtree](https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraExportIT.java#L59).

However, this doesn't at all use the actual types on resources (as, I was uncertain as to how best to undertake this from within the scope of integration testing suites).  I can attempt to implement this by focusing upon identifying the types of the FedoraBaseResource-based objects, but I just want to verify that this is indeed what is being identified as a the optimal approach.

Alternatively, if this refers to resources within the context of Jena Graphs, I can also look to implement this in an approach which more deeply integrates with the Jena API.

Please let me know how best to proceed.